pkgs/applications/audio/zynaddsubfx/default.nix

```diff
@@ -87,29 +95,37 @@ in stdenv.mkDerivation rec {
     ++ lib.optional (guiModule == "fltk") "-DFLTK_SKIP_OPENGL=ON";
 
   doCheck = true;
-  checkInputs = [ cxxtest ];
+  checkInputs = [ cxxtest ruby ];
 
   # TODO: Update cmake hook to make it simpler to selectively disable cmake tests: #113829
   checkPhase = let
-    # Tests fail on aarch64
-    disabledTests = lib.optionals stdenv.isAarch64 [
-      "MessageTest"
-      "UnisonTest"
-    ];
+    disabledTests =
+      # PortChecker test fails when lashSupport is enabled because
+      # zynaddsubfx takes to long to start trying to connect to lash
+      lib.optionals lashSupport [ "PortChecker" ]
+
+      # Tests fail on aarch64
+      ++ lib.optionals stdenv.isAarch64 [ "MessageTest" "UnisonTest" ];
   in ''
     runHook preCheck
     ctest --output-on-failure -E '^${lib.concatStringsSep "|" disabledTests}$'
     runHook postCheck
   '';
 
+  # Use Zyn-Fusion logo for zest build
+  # Manually derived from https://github.com/mruby-zest/mruby-zest-build/blob/3.0.6/src/mruby-zest/example/ZynLogo.qml#L65-L97
+  postInstall = lib.optionalString (guiModule == "zest") ''
+    rm -r "$out/share/pixmaps"
+    mkdir -p "$out/share/icons/hicolor/scalable/apps"
+    cp ${./ZynLogo.svg} "$out/share/icons/hicolor/scalable/apps/zynaddsubfx.svg"
```

Comment:
Please fetchurl the logo

Reply:
There's no SVG logo to fetch, I had to manually write the SVG by using the QML file that draws it in-app as a reference: https://github.com/mruby-zest/mruby-zest-build/blob/3.0.6/src/mruby-zest/example/ZynLogo.qml#L65-L97.

Reply:
😬 then we should add a comment that this question does not come up again.

Reply:
Oh yeah the comment I have there should probably be clearer about why I did that :sweat_smile: